packages/sw-precaching/src/lib/controllers/revisioned-cache-manager.js

```diff
@@ -53,6 +54,21 @@ class RevisionedCacheManager extends BaseCacheManager {
   addToCacheList({revisionedFiles} = {}) {
     assert.isInstance({revisionedFiles}, Array);
     super._addEntries(revisionedFiles);
+
+    const urlsWithoutRevisionFields = revisionedFiles
+      .filter((entry) => typeof entry === 'string');
+    if (urlsWithoutRevisionFields.length > 0) {
+      logHelper.debug({
+        that: this,
+        message: `Some precache entries don't include a separate revision
```

Comment:
I'll cut down on the word count.

But there's nuance here, as this usage might either be legitimate or an error, and I want to provide enough context to help the developer understand what's going on and whether they need to take action.

If I read your proposed message, I'd probably just go to the list of URLs passed to `precache()` (which, maybe I generated by hand, despite the guidance to use `sw-build`) and append `?cacheBust=${Date.now()}` to the end of them.